tap-snapshots/test/lib/commands/audit.js.test.cjs

```diff
@@ -41,6 +41,231 @@ added 1 package, and audited 2 packages in xxx
 found 0 vulnerabilities
 `
 
+exports[`test/lib/commands/audit.js TAP audit signatures ignores optional dependencies > must match snapshot 1`] = `
+verified registry signatures, audited 1 package in 0s
```

Comment:
Do we know why the `0s` here isn't being picked up by the cleanSnapshot?

Reply:
https://github.com/npm/cli/pull/4827/files#r885937015